<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QDialog" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>702</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_main">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="resolution_combo_label">
               <property name="text">
                <string>Resolution:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="resolution_combo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="codec_combo_label">
               <property name="text">
                <string>Codec:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="codec_combo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="scene_combo_label">
               <property name="text">
                <string>Scene:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="scene_combo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="objective_metric_combo_label">
               <property name="text">
                <string>Objective metric:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="objective_metric_combo"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="bitrate_input_label">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string>Bitrate (Mbit):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="bitrate_input"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="loss_input_label">
               <property name="text">
                <string>Packet loss (%):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="loss_input"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="objective_metric_input_label">
               <property name="text">
                <string>Objective value:</string>
               </property>
               <property name="textFormat">
                <enum>Qt::TextFormat::AutoText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="objective_metric_input"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="file_screen_button">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Open a file</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refresh_button">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submit_button">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="result_desc_label">
           <property name="text">
            <string>MOS result (1 - 5):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="result_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_file">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="input_menu_label">
             <property name="text">
              <string>Go back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="home_button">
             <property name="text">
              <string>Input menu</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="results_file_label">
             <property name="text">
              <string>Load file with predicted results</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="load_results_button">
             <property name="text">
              <string>Load results file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="load_inputs_label">
             <property name="text">
              <string>Load file with inputs for prediction</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="load_file_button">
             <property name="text">
              <string>Load inputs file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="save_label">
             <property name="text">
              <string>Save the table to a file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="save_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Save file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="predictions_table">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string>Scene</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Codec</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Resolution</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Bitrate (Mbit)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Packet loss (%)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SSIM (0 - 1)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>VMAF (0 - 100)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MOS (1 - 5)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
